service/service.go

```diff
@@ -114,30 +114,30 @@ func New(options ...Option) (Owner, error) {
 		svc.configProvider = config.Load()
 	}
 
-	svc.setupLogging(svc.configProvider)
+	// Initialize metrics. If no metrics reporters were Registered, do nop
+	// TODO(glib): add a logging reporter and use it by default, rather than nop
+	svc.setupMetrics()
```

Comment:
Shouldn't we setup standard config first(it has service name, owner, all the important things), all metrics and then logger?